gateway/constants.ts

```diff
@@ -0,0 +1,68 @@
+export const GATEWAY_URL = 'wss://gateway.discord.gg/?v=8&encoding=json';
+
+// https://discord.com/developers/docs/topics/gateway#identify-identify-connection-properties
+export const GATEWAY_PROPERTIES = {
+  $os: 'linux',

```

Comment:
Don't hardcode this, use whatever value node provides with `process.platform`

Reply:
Would the reply from the gateway change if we do that?

Reply:
No, it's just discord wanting to know what platform the client is using.

Reply:
so it's not needed to change that

Reply:
True, but it's good practice to not hardcode things.

Reply:
you can try setting it to linux by default but allowing the client to override it with the or operator

Reply:
Would there be any benefit? it does exactly what it needs to do